Library/Formula/aircrack-ng.rb

```diff
@@ -16,11 +14,6 @@ class AircrackNg < Formula
   patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!